src/SDKs/ServiceBus/ServiceBus.Tests/Tests/ScenarioTests.SubscriptionsTests.Detail.CRUD.cs

```diff
@@ -0,0 +1,185 @@
+                            Tier = SkuTier.Standard
+                        }
+                    });
+
+                Assert.NotNull(createNamespaceResponse);
+                Assert.Equal(createNamespaceResponse.Name, namespaceName);
+
+                TestUtilities.Wait(TimeSpan.FromSeconds(5));
+
+                // Create a Topic
+                var topicName = TestUtilities.GenerateName(ServiceBusManagementHelper.TopicPrefix);
+
+                var createTopicResponse = this.ServiceBusManagementClient.Topics.CreateOrUpdate(resourceGroup, namespaceName, topicName,
+                new SBTopic() { EnablePartitioning = true });
+                Assert.NotNull(createTopicResponse);
+                Assert.Equal(createTopicResponse.Name, topicName);
+
+                // Get the created topic
+                var getTopicResponse = ServiceBusManagementClient.Topics.Get(resourceGroup, namespaceName, topicName);
+                Assert.NotNull(getTopicResponse);
+                Assert.Equal(EntityStatus.Active, getTopicResponse.Status);
+                Assert.Equal(getTopicResponse.Name, topicName);
+
+                // Create Invalid TopicName
+                var subscriptionNameGeneration = TestUtilities.GenerateName(ServiceBusManagementHelper.SubscritpitonPrefix);
+                var subscriptionInvalidName = subscriptionNameGeneration + "$";
+
+                //Invalid SubscriptionName 
+                try
+                {
+                    var createinvalidSubResponse = ServiceBusManagementClient.Subscriptions.CreateOrUpdate(resourceGroup, namespaceName, topicName, subscriptionInvalidName,
+                                    new SBSubscription() { MaxDeliveryCount = 14, DeadLetteringOnMessageExpiration = true });
+                }
+                catch (Exception e)
+                {
+                    Assert.Equal("Operation returned an invalid status code 'InternalServerError'", e.Message);
+                }
+
+                // Create Subscription.
+                var subscriptionName = TestUtilities.GenerateName(ServiceBusManagementHelper.SubscritpitonPrefix);
+                SBSubscription createSub = new SBSubscription();
+
+                createSub.EnableBatchedOperations = true;
+                createSub.LockDuration = TimeSpan.Parse("00:03:00");
+                createSub.DefaultMessageTimeToLive = TimeSpan.Parse("00:05:00");
+                createSub.DeadLetteringOnMessageExpiration = true;
+                createSub.MaxDeliveryCount = 14;
+                createSub.Status = EntityStatus.Active;
+                createSub.AutoDeleteOnIdle = TimeSpan.Parse("00:07:00");
+                createSub.DeadLetteringOnFilterEvaluationExceptions = true;
+
+                var createSubscriptionResponse = ServiceBusManagementClient.Subscriptions.CreateOrUpdate(resourceGroup, namespaceName, topicName, subscriptionName, createSub);
+                Assert.NotNull(createSubscriptionResponse);
```

Comment:
All properties values must be verified which are set during create or update

Reply:
Done!